packages/invoices/src/api.ts

```diff
@@ -114,7 +118,10 @@ export const onRead =
 
     return {
       ...invoice,
-      lines: linesGuard(invoice.lines),
+      lines: linesGuard(invoice.lines).map(line => ({
+        ...line,
+        stockLineId: '',
```

Comment:
I added `stockLineId` to `InvoiceLine` and it's not on `InvoiceLineNode`.. 🙄  .. but it is on `OutboundShipmentRow` -  sigh! Needs tidying up, just doing what I have to right now 🤣